### ISSUE TITLE
not clear description

### ISSUE THREAD

**Issue by brakdag** [pranavraja/youtube#1]
it's not clear description how use it.

comandline execution

nodes download.js "path" ???

**Comment by Tornadocraver** [comment]
@brakdag I agree. @pranavraja I would love to use this but cannot find out how.

**Comment by pranavraja** [comment]
Hey guys

This was a little script I wrote several years ago for learning purposes, and didn't have time to clean up/document it much.

Unfortunately it no longer works because the data services it relies on from youtube no longer exist. :(

Nowadays I'd probably recommend something like rg3/youtube-dl instead.
